packages/hash/api/src/seed-data/dev-users.ts

```diff
@@ -32,7 +35,7 @@ const devUsers: readonly DevelopmentUser[] = [
   },
 ] as const;
 
-const devPassword = "password";
+const devDefaultPassword = "password";
 
 export const ensureDevUsersAreSeeded = async ({
```

Comment:
Should we rename this given that it's now run in the production environment?

Reply:
Addressed in 1bf5eba04529c98b28c0822abd5a765e96a05091 by inlining the default value